integrations/terraform-cloud/main.py

```diff
@@ -0,0 +1,115 @@
+from enum import StrEnum
+from typing import Any
+from client import TerraformClient
+from port_ocean.context.ocean import ocean
+from loguru import logger
+from asyncio import gather
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+
+class ObjectKind(StrEnum):
+    WORKSPACE = "workspace"
+    RUN = "run"
+    STATE_VERSION_OUTPUT = "state-version-output"
+    STATE_VERSION = "state-version"
+
+
+def init_terraform_client() -> TerraformClient:
+    """
+    Intialize Terraform Client
+    """
+    config = ocean.integration_config
+    
+    terraform_client = TerraformClient(
+                    config["terraform_host"],
+                    config["terraform_token"],
+                    )
+
+    return terraform_client
+
+
+async def setup_application()->Any:
```

Comment:
we try to not use any as much as possible

Reply:
also there is no need for a whole function for this
just add it to the start event

Reply:
I've place the code for setup_application within start_event now